packages/sitecore-jss-dev-tools/tsconfig-esm.json

```diff
@@ -4,4 +4,12 @@
     "module": "es6",
     "outDir": "dist/esm"
   },
+  "exclude": [
```

Comment:
Do we need to extend `exclude`? we don't do the same in other packages, or it's because of `src/bin`?

Reply:
Yes, because of `src/bin`. Unless we want it in both `esm` and `cjs` directories, the excludes is needed in both files.